### DISCUSSION TITLE
How I wish I could use Tailwind.

### DISCUSSION THREAD

**Discussion by TristanBrotherton** [tailwindlabs/tailwindcss#9964] [Ideas]
I love Tailwind, but I wish when I'm building components in Svelte I wish I could simply create Tailwind styles in my own classes **with modifiers** and line breaks between them:

```
<style>

        .btn {
            @apply 
                rounded 
                py-2
                px-4
                font-bold
                dark:text-red-200;
        }
</style>
```

I just want to be able to do this scoped to the component I'm building (i don't need them anywhere else), so I don't have gigantic class strings that make for unreadable code.

**Comment by yousif-khalil-abdulkarim** [comment]
try using [class variance authority](https://github.com/joe-bell/cva) it helps writing component libraries with tailwindcss

**Comment by pepkin88** [comment]
Check out this example:
https://github.com/jagreehal/svelte-tailwind-scoped-styles
